install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1151,14 +1151,6 @@ ipv6:
   # -- Enable IPv6 support.
   enabled: false
 
-ipvlan:
```

Comment:
Cannot comment on the actual line, but the documentation for the `datapathMode` helm value also needs an update:

https://github.com/cilium/cilium/blob/8f606bafb696076609a01f88204d895576043950/install/kubernetes/cilium/values.yaml.tmpl#L432-L435

Valid values will be `veth` and `lb-only` after this change, AFAIK.

Reply:
Yes, we've discussed that internally with Martynas and Daniel. This is broken already on master in several places today (as evidenced by the above comment). I'll do a follow up PR to fix that.